### PR TITLE
Draft: Update settings page

### PR DESCRIPTION
**Pull request by aza547**

### CODE REVIEW

File: src/renderer/rendererutils.ts [hunk 1]
```diff
@@ -20,5 +20,5 @@ const getVideoResult = (video: any): boolean => {
 
 export {
     getFormattedDuration,
-    getVideoResult,
+    getVideoResult
```
Comment (tdaugaard) [inline src/renderer/rendererutils.ts:23]: Pretty please leave trailing commas, so the next addition doesn't show the `getVideoResult` line as having changed when in fact it wasn't :p

File: src/settings/AudioSettings.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      new ObsAudioDevice('none', '(None: no sound will be recorded)'),
+      new ObsAudioDevice('all', '(All)'),
+      ...audioDevices.output,
+    ]
+  };
+
+  /**
+   * setSetting, why not just use react state hook?
+   */
+  const setSetting = (stateKey: StateToSettingKeyMapKey, value: any) => {
```
Comment (tdaugaard) [inline src/settings/AudioSettings.tsx:39]: Perhaps this should be split out and be shared amongs the tabs?

File: src/settings/ContentSettings.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import FormLabel from '@mui/material/FormLabel';
+
+export default function ContentSettings() {
+
+  return (
+    <div>
+      <FormLabel id="radios" sx={{color: "white"}}>PvE</FormLabel>
+      <Divider color="black" />
+      <FormGroup sx={{ width: '48ch', padding: 1 }} >
+        <FormControlLabel control={<Checkbox defaultChecked style = {{color: "#bb4220", padding: 5}} />} label="Raids" style = {{color: "white"}} />
```
Comment (tdaugaard) [inline src/settings/ContentSettings.tsx:15]: Sounds like we need `categories` to be labeled according to the type of activity, PvE or PvP and allow them to be dynamically shown here.

File: src/renderer/rendererutils.ts (hunk 1 above)
Comment (aza547) [reply src/renderer/rendererutils.ts:23]: That's great advice actually. I've never considered this!

**Pull request closed without merging**